labs/behaviors/focusable.ts

```diff
@@ -68,19 +67,19 @@ export function mixinFocusable<T extends MixinBase<LitElement>>(
       this[updateTabIndex]();
     }
 
-    [privateIsFocusable] = false;
+    [privateIsFocusable] = true;
     [externalTabIndex]: number | null = null;
     [isUpdatingTabIndex] = false;
-    [hasConstructed] = false;
 
     // tslint:disable-next-line:no-any
```

Comment:
We can remove the constructor entirely. `this[isFocusable] = true` won't do anything the `[privateIsFocusable] = true` as the default.

Reply:
Yup, after removing keeps working